dev/Repeater/APITests/SelectionModelTests.cs

```diff
@@ -980,6 +980,65 @@ public void SelectRangeRegressionTest()
             });
         }
 
+        [TestMethod]
+        public void AlreadySelectedDoesNotRaiseEvent()
+        {
+            var testName = "Select(int32 index), single select";
+
+            RunOnUIThread.Execute(() =>
+            {
+                var list = Enumerable.Range(0, 10).ToList();
+
+                var selectionModel = new SelectionModel() {
+                    Source = list,
+                    SingleSelect = true
+                };
+
+                // Single select index
+                selectionModel.Select(0);
+                selectionModel.SelectionChanged += SelectionModel_SelectionChanged;
+                selectionModel.Select(0);
```

Comment:
I'd suggest wrapping the Select calls that you are testing in Verify.DoesNotThrow() calls so that the test output is a little cleaner if this were to fail.

Reply:
According to visual studio `Verify.NoThrow` is not available in project MUXControlsTestApp. Could that potentially be a problem?

Reply:
And it turns out, you can't run the MUXControlsTestApp with Verify.NoThrow. Should we add it now to the MSInterop.cs Verify class?

Reply:
Nah, I don't think its worth it, what you have is sufficient.